### ISSUE TITLE
Black border around notification cards (Black theme)

### ISSUE THREAD

**Issue by luishenriquetp** [elementary/notifications#232]
### What Happened?

I'm using Elementary elementary OS 7.1 Horus . I'm using black theme. When display notifications, it shows a thick black border around the notification card.

### Steps to Reproduce

Open the sistem and wait for any notification

### Expected Behavior

Show notification cards without thick black border

### OS Version

7.x (Horus)

### Software Version

Latest release (I have run all updates)

### Hardware Info

notebook (Dual-Core Intel® Core(TM) i3-6006U CPU @ 2.00GHz - Intel® HD Graphics 520 - 8 GB memory - 1 TB storage (HDD))

**Comment by danirabbit** [comment]
@TomiOhl yeah I don't think that's what the OP is experiencing. What you're showing looks like a Telegram bug: they're sending an image instead of an icon and we're styling it as an image

**Comment by TomiOhl** [comment]
Since it happens with more apps (Chrome, too), I think it might be worth handling it. The issue is that they are pngs with transparent background - no matter if it's an icon or an image, a rectangular shadow is off here anyways.
I can open a separate issue to not spam here though

**Comment by TomiOhl** [comment]
Same in the sound indicator:
![Image](https://github.com/user-attachments/assets/f8459273-5530-48b7-b1ec-135599ee50b7)
So might be an issue with the stylesheet, or just the two use the same display method

**Comment by danirabbit** [comment]
I'm going to close this as incomplete. I can't reproduce this issue and the issue @TomiOhl is experiencing is something else. Please open a separate report about image borders if there isn't already one
